docs/src/core/initial-setup.md

```diff
@@ -1,3 +1,57 @@
 # Initial Setup
 
+Laniakea requires a postgresql database, a D compiler, and the meson
+build tool to get up and running. If you don't already have those
+packages installed, install them now;
+
+$ sudo apt install postgresql-11 postgresql-11-debversion meson ldc libarchive libcurl cmake
```

Comment:
CMake is not required, that could be dropped (the Meson message if it can't find a module is actually pretty misleading)